spring-web/src/test/java/org/springframework/web/filter/RequestLoggingFilterTests.java

```diff
@@ -44,8 +44,96 @@
  */
 public class RequestLoggingFilterTests {
 
-	private final MyRequestLoggingFilter filter = new MyRequestLoggingFilter();
+	private MyRequestLoggingFilter filter = new MyRequestLoggingFilter();
 
+	@BeforeEach
+	public void reset() {
+		filter = new MyRequestLoggingFilter();
+	}
+
+	@Test
+	public void defaultPrefix() throws Exception {
+		final MockHttpServletRequest request = new MockHttpServletRequest("POST", "/hotels");
```

Comment:
```suggestion
		MockHttpServletRequest request = new MockHttpServletRequest("POST", "/hotels");
```

Please make this change to all affected test methods.